kura/distrib/config/kura.build.properties

```diff
@@ -71,6 +71,7 @@ org.eclipse.kura.example.camel.quickstart.version=1.1.200
 org.eclipse.kura.example.can.version=1.0.100
 org.eclipse.kura.example.gpio.version=1.0.200
 org.eclipse.kura.example.driver.sensehat.version=1.0.0
+org.eclipse.kura.wire.devel.component.provider.version=1.0.0-SNAPSHOT
```

Comment:
I am just wondering if `.devel` could be succinctly changed to `develop` for better comprehension